tests/flot.annotations.Test.js

```diff
@@ -0,0 +1,156 @@
+/* global $, describe, it, beforeEach, afterEach, expect, jasmine, spyOn */
+/* jshint browser: true*/
+
+fdescribe('Flot annotations', function () {
+    'use strict';
+    var d1 = [];
+    for (var i = 0; i < 14; i += 0.5) {
+        d1.push([i, Math.sin(i)]);
+    }
+
+    var d2 = [[0, 3], [4, 8], [8, 5], [9, 13], [11, 3]];
+
+    // A null signifies separate line segments
+
+    var d3 = [[0, 12], [7, 12], null, [7, 2.5], [12, 2.5]];
+    var plot;
+    var placeholder;
+    var ctx;
+    beforeEach(function () {
+        placeholder = setFixtures('<div id="test-container" style="width: 200px;height: 100px;border-style: solid;border-width: 1px"></div>')
+        jasmine.clock().install();
+    });
+
+    afterEach(function () {
+        if (plot) {
+            plot.shutdown();
+        }
+        $('#placeholder').empty();
+        jasmine.clock().uninstall();
+    });
+
+    it('should show annotations', function () {
+        let bg = '#009900';
+        let bc = '#FF0000';
+        let font = '12pt';
+        let color =  '#440056';
+        plot = $.plot(placeholder, [ d1, d2, d3 ], {
+            series: {
+                lines: { show: true },
+                points: { show: true, symbol: 'square' },
+                annotations: {
+                    show: true,
+                    location: 'relative',
+                    content: [
+                        {x: 0.5, y: 0.5, label: 'hello world2\nnewline', arrowDirection: 'n', showArrow: true},
```

Comment:
Can these tests be simplified? It doesn't seem like we need all of this configuration just to assert that a canvas method gets called. It would be much easier to understand if they just used one plot, one annotation, and the default annotation options.